app/build.gradle

```diff
@@ -36,6 +36,12 @@ android {
     }
 }
 
+android {
+    lintOptions {
+        abortOnError false
```

Comment:
Why are we doing this?

Reply:
That piece of code is suggested by Android Studio to fix the following issue:
Error: Version 28 (intended for Android Pie and below) is the last version of the legacy support library, so we recommend that you migrate to AndroidX libraries when using Android Q and moving forward.

Reply:
👍 Maybe pop in a `\\ TODO remove this when we migrate to AndroidX`?